articles/connections/passwordless/configure-passwordless-connection.md

```diff
@@ -0,0 +1,137 @@
+
+The **Message** area accepts Markdown and Liquid syntax. If you choose to use Liquid, you can programmatically construct elements of the message. For example, you can reference the `request_language` parameter to change the language of the message:
+
+```text
+{% if request_language contains 'dutch' %}
+ Hier is uw verificatie code: @@password@@
+{% endif %}
+
+{% if request_language contains 'fr-FR' %}
+ Ceci est votre code: @@password@@
+{% endif %}
+```
+
+The following parameters are available to you when defining the message template (regardless of which syntax you use):
+
+| Parameter | Description |
+| - | - |
+| `password` or `code` | The one-time password to send to the user |
+| `phone_number ` | The user's phone number |
+| `application.name` | The name of the application to which the user is logging in |
+| `request_language` | The requested language to be used for the SMS message's content |
+
+5. Adjust settings for your **OTP Expiry** and **OTP Length**.
+
+| Parameter | Description |
+| - | - |
+| OTP Expiry | The amount of time before the one-time password expires |
+| OTP Length | The length of the one-time password sent to the user |
+
+If you choose to extend the OPT expiry time, we recommend increasing the OTP length for security reasons. Otherwise, the malicious party has a longer time frame to try to guess a short code.
+
+6. Disable Sign Ups
+
+If you want to restrict sign-ups and only allow the addition of new users via the API or the Dashboard, click the **Disable Sign Ups** toggle to enable this feature.
+
+When you are done making configuration changes, click **Save**.
+
+## Configure Passwordless using email
+
+1. Navigate to the [Connections > Passwordless](${manage_url}/#/connections/passwordless) page in the [Auth0 Dashboard](${manage_url}/), and enable the Email toggle.
+
+![Enable Email Passwordless](/media/articles/connections/passwordless/connections-passwordless-list.png)
+
+2. Enter the email address you want users to see as the sender in the **From** field, and set the **Subject** line for your email.
+
+![Configure Email Passwordless](/media/articles/connections/passwordless/connections-passwordless-email.png)
+
+::: note
+If you are using a custom template, you must change the **From** value to an address that does *not* use the domain **auth0.com**. Otherwise, the default email template will be sent.
+:::
+
+3. Edit the **Body** of your email. If you have an email template you want to use, you can provide the HTML for it here. Alternatively, you can use Auth0's default template.
+
+The template editor accepts Liquid syntax embedded within the HTML. This allows you to construct parts of your message programmatically. For example, you can reference the `request_language` parameter to change the language of the message:
```

Comment:
Link to same Liquid syntax doc as earlier.
https://auth0.com/docs/email/liquid-syntax

Since this language changing code is included on that page, consider removing it and pointing them to that page to see how to change the language? Maybe do that with the SMS reference to it too?